Ladybird/main.cpp

```diff
@@ -98,14 +98,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto cookie_jar = TRY(Browser::CookieJar::create(*database));
 
+    s_settings = new Browser::Settings();
```

Comment:
The only reason that Settings had a QObject parent was to opt-in to the QObject memory management scheme. If we're going to manage this object manually instead, we should do three things:

1) remove the QObject from its constructor arguments
2) hold it in an AK::OwnPtr instead of a raw pointer
3) Consider if BrowserWindow is doing too many things in its constructor and consider adding a factory function or another patterns to do less things at once. (i.e., in a separate patch)

Reply:
Not sure how to split all things in `BrowserWindow`... Most of this is "boilerplate" code for creating action